editoast/src/main.rs

```diff
@@ -868,25 +916,41 @@ mod tests {
     use tempfile::NamedTempFile;
 
     #[rstest]
-    async fn import_train_schedule_v2(db_pool: Data<DbPool>) {
+    async fn import_export_train_schedule_v2(db_pool: Data<DbPool>) {
         let conn = &mut db_pool.get().await.unwrap();
 
         let changeset = Timetable::changeset();
         let timetable = changeset.create(conn).await.unwrap();
 
+        let original_json = get_trainschedule_json_array();
         let mut file = NamedTempFile::new().unwrap();
-        file.write_all(get_trainschedule_json_array().as_bytes())
-            .unwrap();
+        file.write_all(original_json.as_bytes()).unwrap();
 
+        // Test import
         let args = ImportTrainArgs {
             path: file.path().into(),
             timetable: Some(timetable.id),
         };
-
         let result = trains_import(args, db_pool.clone()).await;
-
         assert!(result.is_ok(), "{:?}", result);
 
+        // Test to export the import
+        let export_file = NamedTempFile::new().unwrap();
```

Comment:
Make it an expect?

Reply:
`unwrap()` is used on all other tests, do you think we should change the whole file ?

Reply:
Yes.